CONTRIBUTING

```diff
@@ -19,16 +19,14 @@ more.
 
 The project maintains the following source code repositories
 
-* http://git.eclipse.org/c/pde/eclipse.pde.ui.git
-* http://git.eclipse.org/c/pde/eclipse.pde.build.git
+* https://github.com/eclipse-pde/eclipse.pde/
 
-This project uses Bugzilla to track ongoing development and issues.
+This project uses Github issues and pull requests to track ongoing development and issues.
 
-* Search for issues: https://eclipse.org/bugs/buglist.cgi?product=PDE
-* Create a new report: https://eclipse.org/bugs/enter_bug.cgi?product=PDE
+* Issues: https://github.com/eclipse-pde/eclipse.pde/issues
+* Pull requests: https://github.com/eclipse-pde/eclipse.pde/pulls
 
-Be sure to search for existing bugs before you create another one. Remember that
-contributions are always welcome!
+Pull requests are always welcome!
```

Comment:
Maybe we should have a more length paragraph on the community like in 
https://github.com/eclipse-platform/#community
or/and we should update https://github.com/eclipse-pde/#contributing accordingly.

Reply:
I will point to https://github.com/eclipse-platform/#community